.agents/skills/tinyengine-dsl-generator/scripts/check_css.py

```
@@ -0,0 +1,249 @@
+                self.warnings.extend(warnings)
+
+            return len(self.errors) == 0
+
+        except subprocess.TimeoutExpired:
+            self.errors.append("postcss command timed out")
+            return False
+        finally:
+            import os
+            try:
+                os.unlink(temp_file)
+            except:
```

Ruff (0.15.17)

[error] Do not use bare `except`

(E722)